packages/jest-console/src/test/index.test.js

```diff
@@ -70,6 +70,7 @@ describe( 'jest-console', () => {
 		} );
 
 		describe( 'lifecycle', () => {
+			/* eslint-disable jest/no-standalone-expect */
```

Comment:
It's granted as being an odd one to test, but [this rule](https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-standalone-expect.md) could be considered valid here. I guess the alternative is to `throw` an error. It kinda feels redundant if `expect` is able to handle this on our behalf, even if technically out of place outside test cases.

Maybe a "Disable reason" rationale could help explain this.

Reply:
Good idea, I will add some details here. It probably could be done by moving some existing comments, it all is tricky.